test/aspect.spec.js

```diff
@@ -47,15 +45,24 @@ describe('Aspect Functions', () => {
     expect(g.state.errors[1].message).to.equal('Concurrent Runs Not Allowed')
   })
   it('should support throwing in onError', async () => {
+    // Use the single error object to avoid 'Unhandled promise rejection' in
+    // some browsers.
+    const theException = new Error('hi from aspect')
```

Comment:
can we use `let` here?

Reply:
Sure I don't see why not.

Reply:
I will wait to see more changes need to be made and push them together.